### PR TITLE
teet-978 removes ics support from meetings

### PR DESCRIPTION
**Pull request by jkorpelin**
PR to remove ics support from meetings

### CODE REVIEW

File: app/backend/src/clj/teet/meeting/meeting_commands.clj [hunk 1]
```diff
@@ -7,7 +7,6 @@
             teet.meeting.meeting-specs
             [teet.util.datomic :as du]
             [teet.meeting.meeting-db :as meeting-db]
-            [teet.meeting.meeting-ics :as meeting-ics]
```
Comment (tatut) [inline app/backend/src/clj/teet/meeting/meeting_commands.clj:10]: fix lint warnings in requires

File: app/backend/src/clj/teet/meeting/meeting_commands.clj [hunk 2]
```diff
@@ -240,43 +240,26 @@
           :type :notification.type/meeting-updated
           :project project-eid}))))))
 
-(defn email-subject-for-type [meeting type]
-  (case type
-    :invitation
-    (str "TEET: koosoleku kutse: " (meeting-model/meeting-title meeting) " / "
-         "TEET: meeting invitation: " (meeting-model/meeting-title meeting))
+(defn email-subject [meeting]
+  (str "TEET: koosolek uuendatud: " (meeting-model/meeting-title meeting) " / "
```
Comment (tatut) [inline app/backend/src/clj/teet/meeting/meeting_commands.clj:244]: move translations to translation files, so we don't need code changes if they need to be updated

File: app/backend/src/clj/teet/meeting/meeting_commands.clj [hunk 3]
```diff
@@ -240,43 +240,26 @@
...
-                                            invitation-or-past-update?)
-           :parts (case invitation-or-past-update?
-                    :past-update
-                    (email-parts-for-past-update meeting meeting-link)
-                    :invitation
-                    (email-parts-for-ical-invitation ics))})]
+           :subject (email-subject meeting)
+           :parts (email-parts meeting-link)})]
 
+    (println "EMail from: " (environment/config-value :email :from))
```
Comment (tatut) [inline app/backend/src/clj/teet/meeting/meeting_commands.clj:262]: remove debug println

File: app/backend/src/clj/teet/meeting/meeting_commands.clj (hunk 2 above)
Comment (jkorpelin) [reply app/backend/src/clj/teet/meeting/meeting_commands.clj:244]: Did we have some beatiful way of using translations in the backend?

Comment (tatut) [reply app/backend/src/clj/teet/meeting/meeting_commands.clj:244]: yes, we have the same style in backend as well

**Review by tatut — APPROVED** [review]

**Pull request merged**